packages/server-wallet/src/wallet/index.ts

```diff
@@ -66,14 +71,15 @@ export type WalletInterface = {
   createChannels(args: CreateChannelParams, amountOfChannels: number): MultipleChannelResult;
   createChannel(args: CreateChannelParams): SingleChannelResult;
   joinChannel(args: JoinChannelParams): SingleChannelResult;
+  joinChannels(channelIds: ChannelId[]): MultipleChannelResult;
   updateChannel(args: UpdateChannelParams): SingleChannelResult;
   closeChannel(args: CloseChannelParams): SingleChannelResult;
   getChannels(): MultipleChannelResult;
   getState(args: GetStateParams): SingleChannelResult;
   syncChannel(args: SyncChannelParams): SingleChannelResult;
 
-  updateChannelFunding(args: UpdateChannelFundingParams): void;
-
+  updateChannelFunding(args: UpdateChannelFundingParams): SingleChannelResult;
+  updateFundingForChannels(args: UpdateChannelFundingParams[]): MultipleChannelResult;
```

Comment:
Could be nice for the naming to be consistent. So maybe `updateChannelFunding`/`updateChannelsFunding` or `updateFundingForChannel`/`updateFundingForChannels`

Reply:
Another option is to have just one api call `updateChannelsFunding` that works for 1 or more channels.